homeassistant/components/light/lutron_caseta.py

```diff
@@ -0,0 +1,105 @@
+"""Support for Lutron Caseta lights."""
+import logging
+
+from homeassistant.components.light import (
+    ATTR_BRIGHTNESS, SUPPORT_BRIGHTNESS, Light)
+from homeassistant.components.light.lutron import to_hass_level
+from homeassistant.components.light.lutron import to_lutron_level
+
+REQUIREMENTS = ['https://github.com/gurumitts/'
+                'pylutron-caseta/archive/v0.2.0.zip#'
+                'pylutron-caseta==v0.2.0', 'paramiko==2.1.2']
+
+_LOGGER = logging.getLogger(__name__)
+
+
```

Comment:
Now I need it to refactor lutron back to platform. I want not do the same with that.